tests/analysis/test_rdf.py

```diff
@@ -0,0 +1,28 @@
+"""Pytest for dynsight.analysis.compute_rdf."""
+
+from pathlib import Path
+
+import MDAnalysis
+import numpy as np
+
+from dynsight.analysis import compute_rdf
+
+
+def test_compute_rdf() -> None:
+    original_dir = Path.cwd()
+    topology_file = original_dir / "tests/systems/coex/test_coex.gro"
+    trajectory_file = original_dir / "tests/systems/coex/test_coex.xtc"
+    expected_bins = original_dir / "tests/analysis/test_rdf/test_bins.npy"
+    expected_rdf = original_dir / "tests/analysis/test_rdf/test_rdf.npy"
+
+    u = MDAnalysis.Universe(topology_file, trajectory_file)
+
+    selection = "type O"
+
+    test_bins, test_rdf = compute_rdf(
+        universe=u, s1=selection, s2=selection, distances_range=[0.0, 5.0]
```

Comment:
When there is a new line for a list of arguments, then they should be formatted with one argument per line.